docs/infrastructure/workers/dynamic-worker-pools.md

```diff
@@ -23,23 +23,63 @@ Octopus Cloud defines the use of the worker as **finished** when it has been idl
 Each worker is provisioned so that they provide your task a sandbox that is completely isolated from our other customers.
 
 ::: info
-Customers cannot make changes to the default worker pool, or the configuration of the workers themselves. The workers are provisioned using our own service, meaning you don't need to configure anything for them to work in your environment.
+Customers cannot add new workers nor make changes to the configuration of the workers. The workers are provisioned using our own service, meaning you don't need to configure anything for them to work in your environment.
 :::
 
+### Types of Dynamic Worker
+
+Each dynamic worker pool can specify the worker image used. As of July 2020, `WindowsDefault` is the default, which is Windows Server Core 2016. `Windows2019` (Windows Server Core 2019) worker images are rolling out. Once the rollout is complete, Windows Server Core 2016 will be marked as deprecated.
+
+Editing a dynamic worker pool allows you to modify the image used.
+
+When an image is marked as deprecated, you will start to see warnings in the Octopus UI, and in the deployment log. After a suitable deprecation period, deployments will start to fail if they target an image that has hit end-of-life.
```

Comment:
```suggestion
When an image is marked as deprecated, you will see warnings in the Octopus UI, and in the deployment log. After a suitable deprecation period, deployments will start to fail if they target an image that has hit end-of-life.
```

Reply:
Also, do we have something more concrete than. "suitable deprecation period"? A month, three months?

Reply:
> Also, do we have something more concrete than. "suitable deprecation period"? A month, three months?

It did say "6 months", but we were hesitant of painting ourselves into a corner. I might check that one with @MJRichardson

Reply:
I'm going to merge away for now, as we're going to be updating this within the week anyway.